src/phoenix/trace/schemas.py

```diff
@@ -74,7 +76,13 @@ class SpanException(SpanEvent):
     """
 
     def __init__(self, timestamp: datetime, message: str):
-        super().__init__(name="exception", message=message, timestamp=timestamp)
+        super().__init__(
+            name="exception",
+            timestamp=timestamp,
+            attributes={
+                EXCEPTION_MESSAGE: message,
```

Comment:
I think if making these unique, we need something like: 
https://github.com/open-telemetry/opentelemetry-python/blob/d8490c5f557dd7005badeb800095cb51b553c98c/opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py#L972
```
 _attributes = {
            "exception.type": exception.__class__.__name__,
            "exception.message": str(exception),
            "exception.stacktrace": stacktrace,
            "exception.escaped": str(escaped),
        }
```

Reply:
Oh nvm